python_modules/dagster/dagster/core/definitions/event_metadata.py

```diff
@@ -9,41 +9,41 @@
 if TYPE_CHECKING:
     from dagster.core.definitions.events import AssetKey
 
-    ParseableMetadataEntryData = Union[
-        "TextMetadataEntryData",
-        "UrlMetadataEntryData",
-        "PathMetadataEntryData",
-        "JsonMetadataEntryData",
-        "MarkdownMetadataEntryData",
-        "FloatMetadataEntryData",
-        "IntMetadataEntryData",
-        "PythonArtifactMetadataEntryData",
-        str,
-        float,
-        int,
-        list,
-        dict,
-    ]
-
-    EventMetadataEntryData = Union[
-        "TextMetadataEntryData",
-        "UrlMetadataEntryData",
-        "PathMetadataEntryData",
-        "JsonMetadataEntryData",
-        "MarkdownMetadataEntryData",
-        "FloatMetadataEntryData",
-        "IntMetadataEntryData",
-        "PythonArtifactMetadataEntryData",
-        "DagsterAssetMetadataEntryData",
-        "DagsterPipelineRunMetadataEntryData",
-    ]
+ParseableMetadataEntryData = Union[
```

Comment:
Is this metadata stuff related to auto-discovery of bare graphs?

Reply:
woah I don't know what happened here. Must've gotten foisted in by accident.